package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pytest-html",
+  "version": "1.10.0",
+  "license": "MPL-2.0",
+  "devDependencies": {
+    "grunt": "^0.4.5",
+    "grunt-contrib-jshint": "~0.10.0",
+    "grunt-contrib-nodeunit": "~0.4.1",
+    "grunt-contrib-qunit": "^1.2.0",
+    "grunt-contrib-uglify": "~0.5.0"
+  },
+  "scripts": {
+    "test": "grunt compass"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pytest-dev/pytest-html.git"
+  },
+  "description": "pytest-html ===========",
+  "bugs": {
+    "url": "https://github.com/pytest-dev/pytest-html/issues"
+  },
+  "homepage": "https://github.com/pytest-dev/pytest-html#readme",
+  "main": "Gruntfile.js",
+  "dependencies": {
+    "grunt-contrib-qunit": "^1.2.0"
```

Comment:
This and the devDependencies appear to be duplicating the installation of dependencies done in the Travis configuration. Is it necessary to have both?